

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m16:50:38  Running with dbt=1.7.18
[0m16:50:38  Registered adapter: firebolt=1.6.3
[0m16:50:38  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:38  
[0m16:50:40  Concurrency: 1 threads (target='default')
[0m16:50:40  
[0m16:50:40  1 of 1 START seed file seed .................................................... [RUN]
[0m16:50:42  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 1.50s]
[0m16:50:42  
[0m16:50:42  Finished running 1 seed in 0 hours 0 minutes and 3.28 seconds (3.28s).
[0m16:50:42  
[0m16:50:42  [32mCompleted successfully[0m
[0m16:50:42  
[0m16:50:42  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m16:50:42  Running with dbt=1.7.18
[0m16:50:42  Registered adapter: firebolt=1.6.3
[0m16:50:42  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:42  
[0m16:50:43  Concurrency: 1 threads (target='default')
[0m16:50:43  
[0m16:50:43  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:50:45  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 1.56s]
[0m16:50:45  
[0m16:50:45  Finished running 1 incremental model in 0 hours 0 minutes and 3.20 seconds (3.20s).
[0m16:50:45  
[0m16:50:45  [32mCompleted successfully[0m
[0m16:50:45  
[0m16:50:45  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m16:50:45  Running with dbt=1.7.18
[0m16:50:45  Registered adapter: firebolt=1.6.3
[0m16:50:45  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:45  
[0m16:50:47  Concurrency: 1 threads (target='default')
[0m16:50:47  
[0m16:50:47  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:50:49  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 1.52s]
[0m16:50:49  
[0m16:50:49  Finished running 1 incremental model in 0 hours 0 minutes and 3.41 seconds (3.41s).
[0m16:50:49  
[0m16:50:49  [31mCompleted with 1 error and 0 warnings:[0m
[0m16:50:49  
[0m16:50:49    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "1e2dbdbd-cfd9-4a74-b936-f033bcb14946",
      "request_id": "4fbc8836-1616-4cfa-81a7-8dfc0d17932b"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m16:50:49  
[0m16:50:49  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
